lib/snappy_ex.ex

```diff
@@ -0,0 +1,63 @@
+defmodule LambdaEthereumConsensus.SnappyEx do
```

Comment:
We should add some tests for this. You can check `test/snappy_test.exs` for the rust NIF's tests.